### ISSUE TITLE
BT-58 (Renewal Number) / BT-57 (Renewal Description); Description and U-AP; mandatory logics

### ISSUE THREAD

**Issue by tinne** [eForms/eForms#99]
Kindly explain the CMs with these two BTs:

- One would explain Renewal Number to default to Zero, making the term optional.
- One would also expect Renewal Description as very optional in the sense that it may only specified when a (positive) Renewal Number is given and would still be optional.

**Comment by JachymHercher** [comment]
Both terms are in fact conditional on BT-56 (Renewal) being "yes". However, you rightly imply that BT-56 is actually not necessary, because the same information is given by BT-58 being 1. The following changes should solve this:

- Removing BT-56.
- Changing the legal status of BT-58 from CM to M.
- Changing the description of BT-58 to:

> "The maximum number of times the contract can be renewed. _By renewing,_ the buyer reserves the right (i.e. not an obligation) to renew the contract (i.e. extend its duration) without a new procurement procedure. For example, a contract may be valid for one year and the buyer may keep a possibility to renew it _(e.g. once, twice)_ for another three months, if he is content with the services he received."

BT-57 should be CM (not O), because the directives require any options to be described (see 2014/24/EU Annex V Part C Point 7). 

(By the way: in the previous round of consultation, we distinguished between "optional" and "very optional". This was removed for simplicity of presentation, but will be implemented as business rules.)
